_publications/alsaadi2020impeccable.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "IMPECCABLE: Integrated Modeling PipelinE for COVID Cure by Assessing Better LEads"
+collection: publications
+permalink: /publications/alsaadi2020impeccable
+date: 2020-10-13
+type: pub
+author: "Aymen AlSaadi, Dario Alfe, Yadu Babuji, Agastya Bhati, Ben Blaiszik, Thomas Brettin, Kyle Chard, Ryan Chard, Peter Coveney, Anda Trifan, Alex Brace, Austin Clyde, Ian Foster, Tom Gibbs, Shantenu Jha, Kristopher Keipert, Thorsten Kurth, Dieter Kranzlmüller, Hyungro Lee, Zhuozhao Li, Heng Ma, Andre Merzky, Gerald Mathias, Alexander Partin, Junqi Yin, Arvind Ramanathan, Ashka Shah, Abraham Stern, Rick Stevens, Li Tan, Mikhail Titov, Aristeidis Tsaris, Matteo Turilli, Huub Van Dam, Shunzhou Wan, David Wifling"
+venue: "Arxiv"
+arxiv: https://arxiv.org/pdf/2010.06574.pdf
```

Comment:
This should point to the paper page, not the pdf. Can you change it? Look at the example on the above comment